package.json

```diff
@@ -42,7 +42,7 @@
     "classnames": "^2.2.6",
     "codemirror": "^5.61.1",
     "express": "^4.17.1",
-    "firebase": "^10.7.1",
+    "firebase": "^11.6.1-firebase-studio-sdk-integration.226be0bb1",
```

Comment:
Are we merging this for an official release?

Reply:
No, we'll wait for the real release - adding a note to the descriptio n